.dockerignore

```diff
@@ -1,6 +0,0 @@
-.git
-.gitignore
```

Comment:
Can you please revert these changes? I believe there was a rebase/merging issue with this branch.

Reply:
Done